components/Image/index.jsx

```diff
@@ -106,7 +108,7 @@ Image.defaultProps = {
   height: 'auto',
   src: '',
   title: '',
-  width: 'auto',
+  width: 'smart',
```

Comment:
Why this change? I'd prefer keeping `auto`, unless there's a reason to use something else:
```suggestion
  width: 'auto',
```

Reply:
Ahhh, [I see now](https://github.com/readmeio/markdown/pull/411#discussion_r779953681). Yeah, I don't think we actually want this to be the default; we *want* them to use the browser's intrinsic `auto` sizing!

Reply:
ok yeah, to your point that we don't want to change this and then have default styles mess up people's sites—going to revert this back to auto from smart